projects/plugins/jetpack/class.jetpack-admin.php

```diff
@@ -273,6 +277,76 @@ static function is_module_available( $module ) {
 
 	}
 
+	/**
+	 * Returns why a module is unavailable.
+	 *
+	 * @param  array $module The module.
+	 * @return string|false A string stating why the module is not available or false if the module is available.
+	 */
+	public static function get_module_unavailable_reason( $module ) {
+		if ( ! is_array( $module ) || empty( $module ) ) {
+			return false;
+		}
+
+		if ( self::is_module_available( $module ) ) {
+			return false;
+		}
+
+		/**
+		 * We never want to show VaultPress as activatable through Jetpack so return an empty string.
+		 */
+		if ( 'vaultpress' === $module['module'] ) {
+			return '';
+		}
+
+		/*
+		 * WooCommerce Analytics should only be available
+		 * when running WooCommerce 3+
+		 */
+		if (
+			'woocommerce-analytics' === $module['module']
+			&& (
+					! class_exists( 'WooCommerce' )
+					|| version_compare( WC_VERSION, '3.0', '<' )
+				)
+			) {
+			return 'Requires WooCommerce 3+ plugin';
```

Comment:
Can we make all those translatable?

Reply:
🤦 Great catch, sorry I missed this. Code-freeze is messing with my brain cells..